### PR TITLE
docs: error pages alignment fix and styling changes on aio-search-results

### PR DESCRIPTION
**Pull request by helloankit**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Alignment issues were there in the heading texts of the views of following pages -

- file-not-found.md
- 404-body.html
- file-not-found-search.component.ts

Also, some minor styling issues were there.

## What is the new behavior?
Styling is consistent and alignment is properly done on the affected pages.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Screenshots
1.  Changes in Alignment
Issue 1

-----------

2. Added alert-box style to the info text
Issue 2

-----------

3. Reduced font-weight, so that text doesn't look blurry.
Issue 3

### CODE REVIEW

File: aio/src/styles/2-modules/search-results/_search-results.scss [hunk 1]
```diff
@@ -53,7 +53,7 @@ aio-search-results {
           padding: 0.5rem 0;
 
           .search-result-item {
-            font-weight: bold;
+            font-weight: 500;
```
Comment (petebacondarwin) [inline aio/src/styles/2-modules/search-results/_search-results.scss:56]: The original aim here is to make the most important search results stand-out. So I am not sure this specific change is desired.

**Review by petebacondarwin — COMMENTED** [review]
Apart from the priority-pages change, these look good to me. Thanks for polishing our website.

Perhaps we need input from dev-rel or the docs team about the search results styling...

File: aio/src/styles/2-modules/search-results/_search-results.scss (hunk 1 above)
Comment (codebriefcase) [reply aio/src/styles/2-modules/search-results/_search-results.scss:56]: @petebacondarwin  I felt that the font-weight was too much and it was making the text blurry. We can revert the font styling for now. I will find a better way to fix it.

Comment (petebacondarwin) [reply aio/src/styles/2-modules/search-results/_search-results.scss:56]: If you defer that change to another PR, I will be happy to approve this one :-)

Comment (codebriefcase) [reply aio/src/styles/2-modules/search-results/_search-results.scss]: @petebacondarwin  Reverted the changes in scss file. Thanks for your quick reply! :)

**Review by petebacondarwin — APPROVED** [review]
Please can you squash the commits into one?
Or alternatively change the 2nd commit to a fixup? See https://github.com/angular/angular/blob/master/docs/FIXUP_COMMITS.md

**Comment by codebriefcase** [comment]
> Please can you squash the commits into one?
> Or alternatively change the 2nd commit to a fixup? See https://github.com/angular/angular/blob/master/docs/FIXUP_COMMITS.md

Done. Please check.

**Review by petebacondarwin — APPROVED** [review]
Great! LGTM

I'm really sorry I just realised that the commit should be `fix(docs-infra); ...` not `docs:` since this is actually a fix to the infrastructure that renders the docs rather than the content of the docs.

Would you please amend the commit one more time?

**Comment by codebriefcase** [comment]
> Great! LGTM
> 
> I'm really sorry I just realised that the commit should be `fix(docs-infra); ...` not `docs:` since this is actually a fix to the infrastructure that renders the docs rather than the content of the docs.
> 
> Would you please amend the commit one more time?

Done

**Review by petebacondarwin — APPROVED** [review]
I rebased on master because the CI config had changed, preventing CI jobs from running.

Reviewed-for: global-docs-approvers

**Comment by petebacondarwin** [comment]
I'm afraid the commit message needs a bit more content. See the lint job failure in CircleCI.
Could you fix that and rebase on master please?

**Review by mgechev — APPROVED** [review]

**Pull request closed without merging**